src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/SNSService.kt

```diff
@@ -29,6 +29,16 @@ class SNSService(
         )
         publish(objectMapper.writeValueAsString(message))
       }
+      ReferralEventType.ASSIGNED -> {
+        val message = mapOf(
+          "eventType" to "intervention.referral.assigned",
+          "description" to "A referral has been assigned to a service user",
+//          "assigned_to" to event.referral.assignedTo,
```

Comment:
Exact values & callback url will be added once referral assignment code is merged